SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1018,6 +1034,10 @@ void USpatialReceiver::RemoveActor(Worker_EntityId EntityId)
 			}
 		}
 	}
+	PendingOwnerOnlyComponents.RemoveAll([&EntityId](const PendingAddComponentWrapper& Queued)
+	{
+		return Queued.EntityId == EntityId;
+	});
```

Comment:
Not entirely sure this is all the cleanup we need for this queue since it isn't auto-cleared in `LeaveCriticalSection` as the others are